rita/src/rita_common/dao_manager/mod.rs

```diff
@@ -169,13 +191,13 @@ fn check_cache(their_id: Identity, entries: &HashMap<Identity, Vec<DAOEntry>>) -
                         their_id.clone(),
                         entry.dao_address
                     );
-                // send approval
+                    send_membership_message(true, their_id.clone());
                 } else if !timer_check(entry.last_updated) {
                     get_membership(entry.dao_address, entry.id.clone());
                 }
             }
             trace!("{:?} is not on any SubnetDAO", their_id);
-            // send rejection
+            send_membership_message(false, their_id);
```

Comment:
Just noticed there is no `.clone()` in `their_id` as there is few lines above. Suspicious.

Reply:
you don't need it since the object is dropped shortly after, should I add it anyway?